jsonnet/kube-prometheus/components/mixin/alerts/tests.yaml

```diff
@@ -1,157 +0,0 @@
-# TODO(metalmatze): This file is temporarily saved here for later reference
-# until we find out how to integrate the tests into our jsonnet stack.
```

Comment:
I see that this file was intended to be removed. Just asking to better understand 😬 

Was this file used to run integration tests in the past? We don't run those tests in CI anymore or was it replaced with something else?

Reply:
No, this is a unit test definition for alerts. Same as https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/master/tests.yaml in kubernetes-mixin.